test/manipulation.test.js

```diff
@@ -901,6 +902,103 @@ describe('manipulation', function () {
         e.should.be.eql(new Error('Invalid date: X'));
       }
     });
+  });
+
+  describe('update|updateAll', function() {
+    beforeEach(function destroyFixtures(done) {
+      Person.destroyAll(done);
+    });
+
+    beforeEach(function createFixtures(done) {
+      var people = [{
+        name: 'foo',
+        age: 19
+      }, {
+        name: 'bar',
+        age: 20
+      }, {
+        name: 'baz',
+        age: 21
+      }, {
+        name: 'qux',
+        age: 22
+      }, {
+        name: 'quux',
+        age: 23
+      }];
```

Comment:
I guess using real names is more important when developing an application using [User-Centered Design](http://en.wikipedia.org/wiki/User-centered_design) and [Personas](http://en.wikipedia.org/wiki/Persona_(user_experience)) and it's less important when developing a framework. I think it's a good habit.